notifications/provision.go

```diff
@@ -30,3 +31,11 @@ func Provision(app, webHookURL string) error {
 	}
 	return nil
 }
+
+func getType(webHookURL string) string {
```

Comment:
I don't like that, the type should be saved in the notification, and read directly from the struct, no logics like that should appear on the client side